packages/render-fidelity-tools/package.json

```diff
@@ -24,8 +24,8 @@
   },
   "dependencies": {
     "@actions/core": "^1.2.4",
-    "@babylonjs/core": "^4.2.0-alpha.26",
-    "@babylonjs/loaders": "^4.2.0-alpha.26",
+    "@babylonjs/core": "^5.0.0-alpha.31",
+    "@babylonjs/loaders": "^5.0.0-alpha.31",
```

Comment:
@augmentedsky I had to update babylon to get it to build with the updated TS version. I have not updated the renders yet, so that'll be your next task.